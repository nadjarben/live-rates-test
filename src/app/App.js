import React from "react"
import LiveRates from "./LiveRates"

function App() {
  return <LiveRates />
}

export default App
