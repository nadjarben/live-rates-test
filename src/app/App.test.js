import React from 'react';
import { shallow } from 'enzyme'
import { Provider } from 'react-redux';
import { store } from '../store';
import App from './App';

describe('App', () => {
  test('Does match snapshots', () => {
    const shallowedApp = shallow(
      <Provider store={store}>
        <App />
      </Provider>
    )
    expect(shallowedApp).toMatchSnapshot()
  });
})

