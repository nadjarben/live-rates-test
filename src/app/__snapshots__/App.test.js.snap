// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Does match snapshots 1`] = `ShallowWrapper {}`;
